ui/jobs/buff_tracker.ts

```diff
@@ -485,29 +510,39 @@ export class BuffTracker {
         cooldown: 120,
       },
       brotherhood: {
+        cooldownAbility: kAbility.Brotherhood,
         gainEffect: EffectId.Brotherhood,
         loseEffect: EffectId.Brotherhood,
         useEffectDuration: true,
+        partyonly: true,
         icon: '../../resources/ffxiv/status/brotherhood.png',
         // Dark Orange.
         borderColor: '#994200',
         sortKey: 11,
         cooldown: 90,
       },
       devotion: {
+        // FIXME: pet is not be considered as inParty, so cannot trace it if missing you.
+        // By the way, pet can delay using devotion after been ordered
+        // and if you order it to continue moving, it can greatly delay upto 30s,
```

Comment:
```suggestion
        // and if you order it to continue moving, it can greatly delay up to 30s,
```

Thanks for the comment.  I think that seems fine to have this limitation on devotion.